psse/psse-converter/src/main/java/com/powsybl/psse/converter/TransformerConverter.java

```diff
@@ -0,0 +1,795 @@
+            regulatingForcedToOff = forceRegulatingToOff(regulatingForcedToOff, regulating);
+        }
+        return regulatingForcedToOff;
+    }
+
+    private static boolean defineVoltageControl(Network network, String id, PsseTransformerWinding winding, RatioTapChanger rtc,
+        double vnom, boolean regulatingForcedToOff) {
+        if (Math.abs(winding.getCod()) == 2) {
+            LOGGER.warn("Transformer {}. Reactive power control not supported", id);
+            return false;
+        }
+        if (Math.abs(winding.getCod()) != 1) {
+            return false;
+        }
+
+        double vmin = winding.getVmi() * vnom;
+        double vmax = winding.getVma() * vnom;
+        double targetV = (vmin + vmax) * 0.5;
+        double targetDeadBand = vmax - vmin;
+        Terminal regulatingTerminal = defineRegulatingTerminal(network, id, winding);
+
+        boolean regulating = true;
+        if (targetV <= 0.0 || targetDeadBand < 0.0) {
+            regulating = false;
+        }
+        if (regulating && regulatingForcedToOff) {
+            LOGGER.warn("Transformer {}. Regulating control forced to off. Only one control is supported", id);
+            regulating = false;
+        }
+        rtc.setTargetV(targetV)
+            .setTargetDeadband(targetDeadBand)
+            .setRegulationTerminal(regulatingTerminal)
+            .setRegulating(regulating);
+
+        return regulating;
+    }
+
+    private static boolean defineActivePowerControl(Network network, String id, PsseTransformerWinding winding, PhaseTapChanger ptc, boolean regulatingForcedToOff) {
+        if (Math.abs(winding.getCod()) != 3) {
+            return false;
+        }
+
+        double activePowerMin = winding.getVmi();
+        double activePowerMax = winding.getVma();
+        double targetValue = 0.5 * (activePowerMin + activePowerMax);
+        double targetDeadBand = activePowerMax - activePowerMin;
+        Terminal regulatingTerminal = defineRegulatingTerminal(network, id, winding);
+        boolean regulating = true;
+        if (targetDeadBand < 0.0) {
+            regulating = false;
+        }
+        if (regulating && regulatingForcedToOff) {
+            LOGGER.warn("Transformer {}. Regulating control forced to off. Only one control is supported", id);
+            regulating = false;
+        }
+
+        ptc.setRegulationValue(targetValue)
+            .setTargetDeadband(targetDeadBand)
+            .setRegulationTerminal(regulatingTerminal)
+            .setRegulationMode(PhaseTapChanger.RegulationMode.ACTIVE_POWER_CONTROL)
+            .setRegulating(regulating);
+
+        return regulating;
+    }
+
+    private static boolean forceRegulatingToOff(boolean regulatingForcedToOff, boolean regulating) {
+        return regulatingForcedToOff || regulating;
+    }
+
+    private static Terminal defineRegulatingTerminal(Network network, String id, PsseTransformerWinding winding) {
```

Comment:
Remote control for phase tap changers and ratio tap changers are not supported in PSSE format?

Reply:
Yes, it is supported. I Fixed the conversion to support it. Unitary test added.